ja/index.html

```diff
@@ -0,0 +1,311 @@
+        font-size: 2.5rem;
+        width: fit-content;
+        margin: 0 auto;
+        padding-top: 15vh;
+        padding-right: 5%; /* visually centered since it's rotated */
+        height: 20vh;
+        transform: rotate(-10deg);
+        -webkit-transform: rotate(-10deg);
+      }
+      article {
+        margin: 20vh auto;
+      }
+      section {
+        margin: 10vh auto;
+      }
+      h1:only-child,
+      h2:only-child {
+        margin: 0;
+      }
+      fieldset[disabled],
+      fieldset[disabled] > * {
+        height: auto;
+        opacity: 1;
+        cursor: default;
+      }
+      .no {
+        color: white;
+        background-color: firebrick;
+        border-color: firebrick;
+        cursor: not-allowed;
+      }
+      .rotate {
+        display: inline-block;
+        transform: rotate(45deg);
+        -webkit-transform: rotate(-45deg);
+      }
+      @media (min-width: 768px) {
+        #language {
+          width: fit-content;
+          float: right;
+        }
+      }
+      @media (min-width: 1280px) {
+        article {
+          max-width: 50vw;
+        }
+      }
+    </style>
+  </head>
+  <body>
+    <figure id="lightdarktoggle" aria-label="ライトモードとダークモードを切り替える">☀️</figure>
+    <main class="container">
+      <h1 class="centered intro">「でも<span class="ai">ChatGPT</span>はこう言ってて……」</h1>
+      <article>
+        <h1>あなたは何かを示そうとして情報源に<mark>AI</mark>を引用したのでここに送られました。</h1>
+      </article>
+      <article>
+        <h2>CharGPT、Claude、Geminiなどの大規模言語モデル（LLM）の応答は事実ではありません。</h2>
+        <p>それらは順序に従ってどんな単語が次に来るのがもっともらしいか予測しています。</p>
+        <p>それらは説得力があるように見える情報を生成することはできますが、その情報は正確でなかったり、信頼できなかったりします。</p>
+      </article>
+      <article>
+        <h2>何千冊もの本を読んだけれど、何をどの本で読んだか思い出せない人を想像してください。</h2>
+        <hr>
+        <p>その人はどんなことが<mark>得意</mark>でしょうか？</p>
+        <fieldset role="group" aria-label="プロンプトの例" disabled>
+          <textarea aria-label="例1: これについてのアイデアをブレインストーミングするのを助けてくれますか？" cols="2" readonly>これについてのアイデアをブレインストーミングするのを助けてくれますか？</textarea>
+          <input type="submit" value="↑" aria-hidden="true" disabled />
+        </fieldset>
+        <fieldset role="group" aria-label="プロンプトの例" disabled>
+          <textarea aria-label="例2: これらに共通するパターンは何ですか？" readonly>これらに共通するパターンは何ですか？</textarea>
+          <input type="submit" value="↑" aria-hidden="true" disabled />
+        </fieldset>
+        <fieldset role="group" aria-label="プロンプトの例" disabled>
+          <textarea aria-label="例3: これを要約してください" readonly>これを要約してください</textarea>
+          <input type="submit" value="↑" aria-hidden="true" disabled />
+        </fieldset>
+        <fieldset role="group" aria-label="プロンプトの例" disabled>
+          <textarea aria-label="例4: これをうまく言い表すにはどうしたらいいですか？" readonly>これをうまく言い表すにはどうしたらいいですか？</textarea>
+          <input type="submit" value="↑" aria-hidden="true" disabled />
+        </fieldset>
+        <hr>
+        <p>その人はどんなことが<mark>苦手</mark>でしょうか？</p>
```

Comment:
English version has `class="no"` but your version does not have that. Is this intentional?
```suggestion
        <p>その人はどんなことが<mark class="no">苦手</mark>でしょうか？</p>
```